app/controllers/api/v1/rents_controller.rb

```
@@ -0,0 +1,34 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      before_action :authenticate_user!
+
+      # Summary: List rents of the user or book
+      def index
+        rents = Rent.where(user_id: params['user_id']) if params['user_id'].present?
+        rents = Rent.where(book_id: params['book_id']) if params['book_id'].present?
+        render_paginated(filtrate(rents), each_serializer: UserRentSerializer)
+      end
+
+      # Summary: Create a rent by a user request
+      def create
+        check_creation_params
+        rent = Rent.create(user: current_user,
+                           book: Book.find(params['book_id']),
+                           from: params['from'],
+                           to: params['to'])
```

A few things:
- It is generally better to build (not create) the Rent and then saving it, instead of just directly creating it. That way you can handle errors more easily. Here for example you could build it one line via the current_user `current_user.rents.build(creation_params)` with creation params obtaining and returning the parameters as it should be. 

- There is no need to do a find for the Book here, you should be able to use the id, and if it fails, let it fail and handle the error correctly.

- After you built the Rent, you can try saving it to the DB and then handle the success or the failure of that.
Something like `if rent.save` `a good response` (created status) else `a bad response` (422 status)